projects/swimlane/ngx-ui/src/lib/components/calendar/calendar.component.ts

```diff
@@ -122,31 +210,138 @@ export class CalendarComponent implements OnInit, ControlValueAccessor {
   _value: Date;
   weeks: Month;
 
+  currentView: View;
+  monthsList: string[] = moment.monthsShort();
+  // today's date
+  current: moment.Moment;
+  startYear: number;
+  _minView: View;
+  _defaultView: View;
+
+  get minView(): View {
+    // default: 'date'
+    return this._minView ? this._minView : 'date';
+  }
+
+  @Input('minView')
```

Comment:
It may not matter but we usually put the `@Input()` above both the getter and setter.